client/landing/gutenboarding/components/domain-picker/suggestion-item-placeholder.tsx

```diff
@@ -6,9 +6,8 @@ import React, { FunctionComponent } from 'react';
 const DomainPickerSuggestionItemPlaceholder: FunctionComponent = () => {
```

Comment:
Rather than having 2 modules that export a component that has, or should have, the same structure, I wonder if it would be better to have a single module that exports a placeholder and "real" version of the component, both of which could use a private "base" component.

Just a thought, no need to change here.